### ISSUE TITLE
Alt title for images

### ISSUE THREAD

**Issue by DomBouman** [closetmouse2/project_directory#5]
You forgot to add Alt title to show the current user what phone that link is linked to.

**Comment by closetmouse2** [comment]
Yes this would make things unclear for blind users, thanks for the input!
